### ISSUE TITLE
Acceptance tests broken for unknown reason

### ISSUE THREAD

**Issue by consideRatio** [jupyterhub/jupyter-server-proxy#349]
It seems that something has changed [since october 14](https://github.com/jupyterhub/jupyter-server-proxy/actions), because our test suite has started [failing without changes to the main branch](https://github.com/jupyterhub/jupyter-server-proxy/actions/runs/3373465868). Maybe a dependency has been updated in a breaking way?

Below is a git diff of Python dependencies as they have changed since october 14.

```diff
--- a/broken.txt  --- TEST RUN NOVEMBER 2
+++ b/working.txt --- TEST RUN OCTOBER 14 (19 days earlier)

-anyio==3.6.2
+anyio==3.6.1
-Babel==2.11.0
+Babel==2.10.3
-exceptiongroup==1.0.0
+
-ipykernel==6.16.2
+ipykernel==6.16.0
-jsonschema==4.17.0
-jupyter-core==4.11.2
+jsonschema==4.16.0
+jupyter-core==4.11.1
-jupyter_client==7.4.4
+jupyter_client==7.4.2
-nbclassic==0.4.7
+nbclassic==0.4.5
-nbconvert==7.2.3
+nbconvert==7.2.1
-notebook==6.5.2
-notebook_shim==0.2.0
+notebook==6.5.1
+notebook-shim==0.1.0
-psutil==5.9.3
+psutil==5.9.2
-pyrsistent==0.19.1
-pytest==7.2.0
+pyrsistent==0.18.1
+pytest==7.1.3
-pytest-html==3.2.0
-pytest-metadata==2.0.4
+pytest-html==3.1.1
+pytest-metadata==2.0.2
-pytz==2022.6
+pytz==2022.4
-terminado==0.17.0
-tinycss2==1.2.1
+terminado==0.16.0
+tinycss2==1.1.1
-tomlkit==0.11.6
+tomlkit==0.11.5
-traitlets==5.5.0
+traitlets==5.4.0
-zipp==3.10.0
+zipp==3.9.0

-robotframework-6.0
+robotframework-5.0.1
-robotframework-jupyterlibrary-0.4.1
+robotframework-jupyterlibrary-0.4.0
-
+exceptiongroup-1.0.0rc9
```

@bollwyvl do you have time to look into this as a person knowledgable about the acceptance tests and [robotframework-jupyterlibrary](https://github.com/robots-from-jupyter/robotframework-jupyterlibrary)?

```
Element with locator 'css:.jp-LauncherCard-label[title^="foo"]' not found.
```

That is something that seems [associated with jupyterlab](https://github.com/jupyterlab/jupyterlab/search?q=jp-LauncherCard-label), but I don't get whats going on well enough to go much further.

**Comment by consideRatio** [comment]
https://github.com/jupyterhub/jupyter-server-proxy/pull/348#issuecomment-1299379187:

> `jupyter_core` might not pick up `jupyter_*_config.json` in their previous cwd locations... https://discourse.jupyter.org/t/security-advisory-for-jupyter-core-package/16527.
> 
> `--config-path` might work.

I've failed to figure out how to pass a `--config` flag to the started server though =/

**Comment by consideRatio** [comment]
Closed by #348
